doc/integrator/api.rst

```diff
@@ -34,3 +34,9 @@ In the file ``geoportal/demo_geoportal/static-ngeo/api/index.js`` you can custom
    config.queryableLayers = [<layer a>, <layer b>, ...];
 
 The API help is in the folder ``geoportal/<package>_geoportal/static/apihelp/``.
+
+To be able to have a layer in the api, the layer should be in the themes::
```

Comment:
Should this read:
"In order for a layer to be accessible via API, the layer must be in a group"

(below we are talking about "group", not "theme")

Reply:
in the themes (imply that the group should already be in a visible theme) or do you have a better proposition?